pkg/gatewayserver/io/udp/firewall_ratelimit.go

```diff
@@ -0,0 +1,94 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package udp
+
+import (
+	"sync"
+	"time"
+
+	"go.thethings.network/lorawan-stack/pkg/errors"
+	encoding "go.thethings.network/lorawan-stack/pkg/ttnpb/udp"
+)
+
+var (
+	// defaultMessages is the default number of messages to use for rate limiting.
+	defaultMessages = 10
+
+	// defaultThreshold is the duration after which a new packet will be accepted.
+	defaultThreshold = 10 * time.Millisecond
+)
```

Comment:
These are probably not needed

Reply:
Indeed, they can be removed. GS has default config already